script/script.go

```diff
@@ -385,12 +422,16 @@ func setRequestValue(f filters.FilterContext) func(*lua.LState) int {
 		case "url":
 			u, err := url.Parse(s.ToString(-1))
 			if err != nil {
-				s.Push(lua.LString(err.Error()))
-				return 1
+				s.RaiseError("%v", err)
+				return 0
 			}
 			f.Request().URL = u
+		case "url_path":
+			f.Request().URL.Path = s.ToString(-1)
+		case "url_raw_query":
+			f.Request().URL.RawQuery = s.ToString(-1)
 		default:
-			// do nothing for now
+			s.RaiseError("unsupported request field %s", key)
```

Comment:
this could break production for people, not?

Reply:
This will break if people have assignment to non-existing request property.
I am not sure how to asses such a risk, could be reverted in worst case.

Reply:
I would prefer to comment the code and add a `TODO(sszuecs): <link>` with link to the comment here.

Reply:
We can change this in the v1 branch which is doing the breaking change